Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,267 @@
+                        double number2 = double.Parse(argument2);
+                        double result = 0.0;
+
+                        switch (operatorType)
+                        {
+                            case OperatorType.Addition:
+                                result = number1 + number2;
+                                break;
+                            case OperatorType.Subtraction:
+                                result = number1 - number2;
+                                break;
+                            case OperatorType.Multiplication:
+                                result = number1 * number2;
+                                break;
+                            case OperatorType.Division:
+                                // Doubles can divide by zero
+                                result = number1 / number2;
+                                break;
+                            default:
+                                Console.WriteLine("What in the world happened?");
+                                break;
+                        }
+                        Console.WriteLine($"Result = {result}");
+
+                    }
+                    else if (argument1Type == ArgumentType.String)
+                    {
+                        if (argument2Type == ArgumentType.String)
+                        {
+                            // String + String
+                            if (operatorType == OperatorType.Addition)
+                            {
+                                validExpression = true;
+                                string result = argument1 + argument2;
+                                Console.WriteLine($"Result = {result}");
+                            }
+                            // String - String
+                            else if (operatorType == OperatorType.Subtraction)
+                            {
+                                validExpression = true;
+                                var result = new StringBuilder();
+                                var arg1CharArray = argument1.ToCharArray();
+                                foreach (char arg1Char in arg1CharArray)
+                                {
+                                    if (!argument2.Contains(arg1Char))
+                                    {
+                                        result.Append(arg1Char);
+                                    }
+                                }
+                                Console.WriteLine($"Result = {result}");
+                            }
+                        }
+                        // String * Number
+                        else if (argument2Type == ArgumentType.Number && operatorType == OperatorType.Multiplication)
+                        {
+                            validExpression = true;
+                            var result = new StringBuilder();
+                            var times = (int)double.Parse(argument2);
```

Comment:
This might be a good place for a comment, to explain why you're casting a `double` to an `int`.